src/time_integration/paired_explicit_runge_kutta/methods_PERK2.jl

```diff
@@ -0,0 +1,362 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+#! format: noindent
+
+# Abstract base type for both single/standalone and multi-level P-ERK time integration schemes
+abstract type PERK end
+# Abstract base type for single/standalone P-ERK time integration schemes
+abstract type PERKSingle <: PERK end
+
+function compute_a_coeffs(num_stage_evals, se_factors, mon_coeffs)
+    a_coeffs = mon_coeffs
```

Comment:
```suggestion
    a_coeffs = copy(mon_coeffs)
```
I just realized that this is actually the behaviour we want  - otherwise, we could also try to do this in-place - although there is no real need for this as it is not performance-critical.